bin.src/make_flats.py

```diff
@@ -54,4 +54,6 @@
     my_flat = desc.imsim.make_flat(gs_det, counts_per_iter, niter, rng,
                                    logger=logger, wcs=wcs)
     ccd_id = "R{}_S{}".format(det_name[2:5:2], det_name[8:11:2])
-    my_flat.write('flat_{}_{}_{}.fits'.format(visit, ccd_id, obs_md.bandpass))
+    prefix = config['persistence']['eimage_prefix']
+    my_flat.write('{}_{}_{}_{}.fits'.format(prefix, visit, ccd_id,
+                                            obs_md.bandpass))
```

Comment:
It looks like this makes it so Flats are not differently labeled than normal e-image files.  Is that the intention?

Reply:
OK, git it. I see.  I guess at least the visit number will be different.  Maybe we could have some convention for flat visit # sequences in order to be able to quickly guess from the file name?  AIso, I suppose they typically won't show up in the same directories.

Reply:
I proposed a convention in the ImageProcessingPipeline wiki in the [Generating Calibration Products](https://github.com/LSSTDESC/ImageProcessingPipelines/wiki/Generating-Calibration-Products) entry: `3xxxxxx`, `4xxxxxx`, `5xxxxxx` for biases, darks, and flats, respectively.